gwm_create_service_account.py

```diff
@@ -305,6 +305,13 @@ async def download_service_account_key():
   await retryable_command(command)
 
 
+async def delete_key():
+  input("\nPress any key to continue...")
```

Comment:
Maybe "Press any key after you've downloaded the file", to make it clear? (in case the download UI takes a while to appear, and this prompt appears before it)

Reply:
Done.